src/core/pairwise_matrix_pair.cpp

```diff
@@ -61,8 +63,25 @@ void Matrix::Pair::read(const Index& index) const
         _rawIndex = clusterIndex;
         readNext();
     }
+    return clusterIndex;
+
 }
+/*!
```

Comment:
Please make sure that there are three newlines between each function for readability.

Reply:
Thanks Ben for the check! I fixed the spacing inconsistency.